src/words.ts

```diff
@@ -45,5 +45,5 @@ function stripSpecialCharacters(str: string) {
   // match special symbols and replace with ' '
   str = str.replace(/[.,/#!$%&*;:{}=\-_'"~()]/g, ' ');
   // match double whitespace with single space for cleaner string
-  return str.replace(/\s{2,}/g, ' ');
+  return str.replace(/\s{2,}/g, ' ').split(' ');
```

Comment:
I feel like the split should be on line 22

```
 ...new Set(stripSpecialCharacters(message.content).split(' ')),
 ```
 
 What do you think?

Reply:
👍  That would be better since the function would return what it's supposed to